### PR TITLE
Updated the localization example.

### PR DESCRIPTION
**Pull request by shb46**

### CODE REVIEW

**Comment by dakota002** [comment]
Hey @shb46, can you make a couple changes:

- The example files do not need $Id, that line can be deleted
- The $schema field should point to the schema file that this is meant to be validated against, I assume in this case that means either the Orbit or Pointing schema? This one is a good example of what I mean: https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/icecube/GoldAndBronze.example.json

File: gcn/notices/core/Localization.example.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/core/Localization.schema.json",
```
Comment (dakota002) [inline gcn/notices/core/Localization.example.json:2]:
```suggestion
```

File: gcn/notices/core/Localization.example.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```
Comment (dakota002) [inline gcn/notices/core/Localization.example.json:3]:
```suggestion
  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/core/Pointing.schema.json",
```

File: gcn/notices/core/Localization.example.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/core/Pointing.schema.json",
+  "ra": 0,
+  "dec": 0,
+  "uncertainty_shape": "rectangle",
+  "ra_axis": [-5.5, 7.5],
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Localization.example.json:6]:
```suggestion
  "ra_uncertainity": [-5.5, 7.5],
```

File: gcn/notices/core/Localization.example.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  "dec_axis": [-2.1, 3.3],
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Localization.example.json:7]:
```suggestion
  "dec_uncertainty": [-2.1, 3.3],
```

**Comment by dakota002** [comment]
We added this example in another PR

**Pull request closed without merging**